src/js/index.tsx

```diff
@@ -25,10 +25,10 @@ const App = () => {
   useEffect(() => {
     if (lang === 'en' || lang === 'fr') {
       i18n.changeLanguage(lang);
-    } else {
-      navigate(`/${i18n.language}`);
+    } else if (i18n.language) {
+      navigate(`/${i18n.language}/`);
     }
```

Comment:
should this have a fallback else branch to navigate to `/en/`? otherwise this looks good to me